caesar_cipher/spec/cipher_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'cipher'
+
+describe Cipher do
+  describe '.encrypt' do
+    context " 'Awesome' shift 10 times " do
```

Comment:
@zzokki81 it's great that you added "custom" examples, but you should consider them as additional examples. Leave them here, but it's usually also required to have specs for examples that were mentioned in the assignment/task you get from the client - like the one I've added to the assignment file. You want those specs to pass too.

Reply:
@zzokki81 what about adding specs for the example given in the assignment?